core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -465,8 +465,8 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_O3_LAST_PARTITION_MAX_SPLITS("cairo.o3.last.partition.max.splits"),
     CAIRO_O3_PARTITION_SPLIT_MIN_SIZE("cairo.o3.partition.split.min.size"),
     DEBUG_WAL_PURGE_WAIT_BEFORE_DELETE("debug.wal.purge.wait.before.delete", false, true),
-    WRITER_MEMORY_LIMIT("cairo.writer.memory.limit"),
-    RSS_MEMORY_LIMIT("cairo.rss.memory.limit"),
+    RAM_USAGE_LIMIT_MIB("ram.usage.limit.mib"),
```

Comment:
nit: for other props we ask for bytes, not for MiB. Also, `getLongSize()` supports suffixes, so users may write `ram.usage.limit.mib=128K` which will implicitly mean 128GB - that's weird. It'd be more consistent with other props to name this one `ram.usage.limit.bytes` or `ram.usage.limit` and accept bytes.